tests/acceptance/features/cliCommands/backupConsistency.feature

```diff
@@ -23,4 +27,14 @@ Feature: backup consistency
     And the administrator has stopped the server
     When the administrator checks the backup consistency using the CLI
     Then the command should be successful
+    And the command output should contain "💚 No inconsistency found. The backup in '%storage_path%' seems to be valid."
+
+  @issue-9498
+  Scenario: check backup consistency after uploading a file multiple times
```

Comment:
```suggestion
  Scenario: check backup consistency via CLI command after uploading a file multiple times
```

Reply:
`via CLI command` in a scenario title is rather unnecessary in this feature context (CLI feature context)